drivers/docker/driver_windows.go

```diff
@@ -15,3 +19,19 @@ func getPortBinding(ip string, port string) docker.PortBinding {
 func tweakCapabilities(basics, adds, drops []string) ([]string, error) {
 	return nil, nil
 }
+
+var containerAdminErrMsg = "running container as ContainerAdmin with is unsafe; change the container user, set task configuration to privileged or disable windows_allow_insecure_container_admin to disable this check"
```

Comment:
I think we want to enable the flag to disable the check?

```suggestion
var containerAdminErrMsg = "running container as ContainerAdmin with is unsafe; change the container user, set task configuration to privileged or enable windows_allow_insecure_container_admin to disable this check"
```

Reply:
ugh, sorry, yes. Also there was another typo there. I fix.